.github/workflows/buildvue.yml

```
@@ -0,0 +1,137 @@
+name: Build Vue files
+
+on:
```

Might make sense to make it possible to manually trigger that action for a branch maybe:

```suggestion
on:
  workflow_dispatch:
```

---

I don't think this makes sense, since right now it will always run. I think it's better not to have to think about when it should be run? Especially since this is a safeguard for when people have already forgotten. If the command doesn't work for some reason, we can always build manually until its fixed.